crates/vm/levm/src/gas_cost.rs

```diff
@@ -0,0 +1,688 @@
+use bytes::Bytes;
+/// Contains the gas costs of the EVM instructions (in wei)
+use ethereum_rust_core::{Address, H256, U256};
+
+use crate::{
+    call_frame::CallFrame,
+    constants::{COLD_STORAGE_ACCESS_COST, WORD_SIZE},
+    errors::OutOfGasError,
+    vm::VM,
+    StorageSlot,
+};
```

Comment:
As a general rule, keep the imports together.
```suggestion
use bytes::Bytes;
use ethereum_rust_core::{Address, H256, U256};
use crate::{
    call_frame::CallFrame,
    constants::{COLD_STORAGE_ACCESS_COST, WORD_SIZE},
    errors::OutOfGasError,
    vm::VM,
    StorageSlot,
};
```